package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "dependencies": {
```

Comment:
You're probably right here, but because we need to build plantingjs here, we need also this dependencies. I believe when we will create proper npm package (that one with all kind of scripts) we will be able to remove lot of these dependencies.

And as for dev dependencies - I will create separate issue for that - we will do it in less busy time.